recipe-box-backend/src/main/java/com/endava/recipebox/controller/RecipeController.java

```diff
@@ -2,16 +2,17 @@
 
 
 import com.endava.recipebox.dto.RecipeAddRequestDTO;
+import com.endava.recipebox.dto.RecipeEditRequestDTO;
 import com.endava.recipebox.model.MealType;
 import com.endava.recipebox.dto.RecipeDTO;
 import com.endava.recipebox.service.RecipeService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.*;
 
 
+import java.nio.file.AccessDeniedException;
```

Comment:
remove not used import.
Before commit, you can try on each edited class to do:
In IntelliJ IDEA, you can use the following shortcuts:

1. Remove Unused Imports
Windows/Linux: Ctrl + Alt + O
Mac: ⌘ + Option + O
This shortcut will optimize your imports by removing any that are not used in the current file.

2. Auto-Alignment and Reformatting Code
Windows/Linux: Ctrl + Alt + L
Mac: ⌘ + Option + L